js/foundation.util.touch.js

```diff
@@ -140,15 +140,10 @@ class Touch {}
   };
 
   Touch.init = function($) {
```

Comment:
I think for this specific module an object literal makes more sense, because it doesn't seem like you ever actually instantiate the class itself, unless I'm mistaken.

In cases where it is a class, the `static` keyword works out of the box for methods. If you want static *properties*, you need to add the [transform-class-properties](https://babeljs.io/docs/plugins/transform-class-properties/) plugin.